modules/RTC/TraceablePeerConnection.js

```diff
@@ -1773,6 +1773,15 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(
         description = normalizePlanB(description);
     }
 
+    // Safari WebRTC errors when no supported video codec is found in the offer.
+    // To prevent the error, inject H264 into the video mLine.
+    if (RTCBrowserType.usesNewGumFlow() && RTCBrowserType.isSafari()) {
```

Comment:
Does Safari ever not use the new GUM flow?

Reply:
Safari isn't part of new gum flow at all right now, waiting on only-audio-only to go in first. It's likely Safari 11 and up will be put into new gum while older version are left with temasys for now.